Branch-SDK/Branch-SDK/Branch.h

```diff
@@ -1334,4 +1365,19 @@ typedef NS_ENUM(NSUInteger, BranchPromoCodeUsageType) {
  */
 - (void)log:(NSString *)log;
 
+/**
+ Method used by BranchUniversalObject to register a view on content
+ 
+ @warning This is meant for use internally only and should not be used by apps.
+ */
+- (void)registerViewWithParams:(NSDictionary *)params andCallback:(callbackWithParams)callback;
+
+/**
+ Gets the global Branch instance, whether live or test, but only if a key has been specified.
+ 
+ @warning This is meant for use internally only and should not be used by apps. Invoking this method could result
+ in broken functionality.
+ */
++ (Branch *)getCurrentInstanceIfAny;
+
```

Comment:
How would you reference it from `BranchUniversalObject` then? At the very least, it's unlikely a dev would find it since I named it `getCurrentInstance...`, not `getInstance...`